### PR TITLE
fix: sample test tear down resource on operation

### PR DESCRIPTION
**Pull request by morgandu**
Fixes <https://github.com/googleapis/java-aiplatform/issues/160> ☕️

### CODE REVIEW

File: samples/snippets/src/main/java/aiplatform/ImportDataVideoActionRecognitionSample.java [hunk 1]
```diff
@@ -68,6 +68,11 @@ static void importDataVideoActionRecognitionSample(
       OperationFuture<ImportDataResponse, ImportDataOperationMetadata> response =
           client.importDataAsync(name, importConfigs);
 
+      while (!response.isDone()) {
```
Comment (kurtisvg) [inline samples/snippets/src/main/java/aiplatform/ImportDataVideoActionRecognitionSample.java:71]: Should this be in the sample, or as part of the test once the sample is done?

File: samples/snippets/src/main/java/aiplatform/ImportDataVideoActionRecognitionSample.java [hunk 2]
```diff
@@ -68,6 +68,11 @@ static void importDataVideoActionRecognitionSample(
...
+        System.out.println("Waiting for response...");
+        Thread.sleep(10000);
```
Comment (kurtisvg) [inline samples/snippets/src/main/java/aiplatform/ImportDataVideoActionRecognitionSample.java:73]: Please use the `java.time` package instead.

**Review by kurtisvg — CHANGES_REQUESTED** [review]
I'm not sure what this is accomplishing - the commit says `sample test tear down resource on operation` but I don't see anything being torn down?

**Review by telpirion — APPROVED** [review]
This looks good, per the instructions given to me by @chingor13 .

File: samples/snippets/src/main/java/aiplatform/ImportDataVideoActionRecognitionSample.java (hunk 2 above)
Comment (munkhuushmgl) [reply samples/snippets/src/main/java/aiplatform/ImportDataVideoActionRecognitionSample.java:73]: at line 81, it is already doing the same thing. No need to add while loop in the sample

https://github.com/googleapis/java-aiplatform/blob/5a0b004dfecf78a6eea6f98ccf7ef559f21440cd/samples/snippets/src/main/java/aiplatform/ImportDataVideoActionRecognitionSample.java#L81

**Pull request closed without merging**